### ISSUE TITLE
Installation Issue

### ISSUE THREAD

**Issue by valentinpi** [msys2/msys2-texlive#14]
Hi! I did not know where to put it but when installing the package I get the following error in the end:
```
(3/5) Updating TeXLive filename database...
>>> Running mktexlsr
>>> Running mtxrun (this might take a while)
/
(4/5) Updating TeXLive format files...
>>> Running fmtutil (this might take a while)
fmtutil [ERROR]: running `luajithbtex -ini   -jobname=luajithbtex -progname=luajithbtex luatex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `luatex -ini   -jobname=luatex -progname=luatex luatex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `luahbtex -ini   -jobname=luahbtex -progname=luahbtex luatex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `eptex -ini   -jobname=eptex -progname=eptex *eptex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `euptex -ini   -jobname=euptex -progname=euptex *euptex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `luajittex -ini   -jobname=luajittex -progname=luajittex luatex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: not building aleph due to missing engine: aleph
fmtutil [ERROR]: running `xetex -ini   -jobname=xetex -progname=xetex -etex xetex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `pdftex -ini   -jobname=pdftex -progname=pdftex -translate-file=cp227.tcx *pdfetex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `pdftex -ini   -jobname=pdfetex -progname=pdfetex -translate-file=cp227.tcx *pdfetex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `pdftex -ini   -jobname=etex -progname=etex -translate-file=cp227.tcx *etex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
fmtutil [ERROR]: running `luatex -ini   -jobname=dviluatex -progname=dviluatex dviluatex.ini <nul' return status: 1
fmtutil [ERROR]: returning error due to option --strict
C:\msys64\mingw64\bin\runscript.tlu:915: command failed with exit code 12:
perl.exe c:\msys64\mingw64\share\texmf-dist\scripts\texlive\fmtutil.pl  --sys --all --cnffile mingw64/etc/texmf/web2c/fmtutil.cnf
error: command (/usr/share/libalpm/scripts/mingw-w64-x86_64-texlive-fmtutil.script /usr/share/libalpm/scripts/mingw-w64-x86_64-texlive-fmtutil.script ) failed to execute correctly
(5/5) Updating TeXLive font maps...
>>> Running updmap (this might take a while)
```
In the MSYS2 shell I executed:
```
$ pacman -S mingw-w64-x86_64-texlive-full
```

**Comment by valentinpi** [comment]
Hi! For context: I have texlive by MSYS2 on my main desktop and on my work laptop, the output is from my work laptop where MSYS2 is installed in `C:\msys64`, on my work machine it is in `D:\msys64`, but I had the same error there.

I tried the command, but the first path did not seem to work in the MSYS2 shell. See the file for the output of a slightly different command (note that I'm in the root `C:\msys64` folder).
[tl-log.txt](https://github.com/msys2/msys2-texlive/files/7070576/tl-log.txt)

**Comment by naveen521kk** [comment]
Not sure what's wrong. From your logs it says,
```
! Missing number, treated as zero.
<to be read again>
\chardef
<to be read again>
\rhm@arabic
\addlanguage ...\chardef \csname rhm@#1\endcsname
                                                  =#5 \ifdefined \directlua ...

l.48 \addlanguage{arabic}{zerohyph.tex}{}{}{}

?
! Emergency stop.
<to be read again>
\chardef
<to be read again>
\rhm@arabic
\addlanguage ...\chardef \csname rhm@#1\endcsname
                                                  =#5 \ifdefined \directlua ...

l.48 \addlanguage{arabic}{zerohyph.tex}{}{}{}

!  ==> Fatal error occurred, no output PDF file produced!
```
Something similar was repeating a lot of time.

**Comment by naveen521kk** [comment]
I think I found the problem. In normal texlive installation, I find `language.def` as
```
\addlanguage{farsi}{zerohyph.tex}{}{2}{3}
```
but in msys2 I find it as
```
\addlanguage{farsi}{zerohyph.tex}{}{}{}
```
I remember to ask help on this in [mailing list](https://tug.org/pipermail/tex-live/2021-May/046962.html), but I only fixed the problem partly https://github.com/msys2/msys2-texlive/commit/25216253bd918e6019c1572ec3feebb6013cd280. I will fix it soon.

Thanks for reporting :)
